src/router_core/modules/mobile_sync/mobile.c

```diff
@@ -336,7 +336,7 @@ static qd_message_t *qcm_mobile_sync_compose_absolute_mau(qdrm_mobile_sync_t *ms
         //
         if (qcm_mobile_sync_addr_is_mobile(addr)
             && (DEQ_SIZE(addr->rlinks) > 0
-                || DEQ_SIZE(addr->subscriptions) > 0
+                || (DEQ_SIZE(addr->subscriptions) > 0 && addr->propagate_local)
```

Comment:
Does this change the visibility of $management?

Reply:
It does not.  $management is a subscription on every router and the forwarding treatment is "CLOSEST".  All traffic to $management is always directed to the agent on the ingress router.